cupyx/jit/_internal_types.py

```diff
@@ -73,3 +73,17 @@ def __init__(self):
 
     def __call__(self):
         raise RuntimeError('Cannot call this function from Python layer.')
+
+
+def method_as_builtin_func(instance_name, method):
```

Comment:
We are planning to implement `Callable` type as a subclass of `_cuda_types.TypeBase` and may erase this function in a process of a cosmetic change in the future.

Reply:
Sounds good! Will BuiltinFunc be removed too? I think in this case it is still useful to use BuiltinFunc because they are from CUDA. (btw, this function could have been made as a classmethod... Look a bit nicer that way, so I will change it regardless of the plan for Callable.)

Reply:
Done in 6eef9f3.

Reply:
BuiltinFunc will not be removed